api/src/routes/account.ts

```diff
@@ -0,0 +1,57 @@
+import express, { Router, Request, Response, NextFunction } from 'express';
+const router: Router = express.Router();
+
+import validator from 'validator';
+import jwt from 'jsonwebtoken';
+const secret: any = process.env.JWT_SECRET;
+
+const Account = require('../models/account');
+
+router.post('/create', (req: Request, res: Response, next: NextFunction) => {
+  if (!validator.isEmail(req.body.email)) {
+    res.status(400).send('email is not valid');
+    return;
+  }
+  // username/password validation?
+
+  Account.create({
+    email: req.body.email,
+    username: req.body.username,
+    password: req.body.password,
```

Comment:
Like you said in the comment we should probably add some validation here. Maybe add a follow up task?

Reply:
i.e. making sure password is more than 5 chars, has special characters etc..